projects/hal-components/src/lib/components/popup-connected/popup-connected.component.spec.ts

```diff
@@ -59,6 +67,56 @@ describe('PopupConnectedComponent', () => {
     });
   });
 
+  // Small
+  it('should not be small by default, close-button should exist, should not have class small-popup', () => {
+    expect(fixture.debugElement.queryAll(By.css('.small-popup')).length).toBe(0);
+    expect(fixture.debugElement.queryAll(By.css('.close-button')).length).toBe(1);
+  });
+  describe('Setting small to true', () => {
+    beforeEach(() => {
+      testComponent.small = true;
+      fixture.detectChanges();
+    });
+    it('close-button should not exist, should have class small-popup', () => {
+      expect(fixture.debugElement.queryAll(By.css('.small-popup')).length).toBe(1);
+      expect(fixture.debugElement.queryAll(By.css('.close-button')).length).toBe(0);
+    });
+  });
+  describe('Setting small to false', () => {
+    beforeEach(() => {
+      testComponent.small = false;
+      fixture.detectChanges();
+    });
+    it('close-button should exist, should not have class small-popup', () => {
+      expect(fixture.debugElement.queryAll(By.css('.small-popup')).length).toBe(0);
+      expect(fixture.debugElement.queryAll(By.css('.close-button')).length).toBe(1);
+    });
+  });
+
+  // RelativePositionY
+  it('arrow-above should exist', () => {
+    expect(fixture.debugElement.queryAll(By.css('.arrow-above')).length).toBe(1);
+  });
```

Comment:
Bør flytte opp denne it'n slik at alle decriber er sammen og alle it er sammen på samme nivå